app/assets/stylesheets/application.scss

```diff
@@ -24,17 +24,20 @@
 
 /* Base styles (regardless of theme) */
 .bs-callout {
-margin: 20px 0;
-padding: 15px 30px 15px 15px;
-border-left: 5px solid #eee;
+  padding: 20px;
```

Comment:
Properties should be ordered border, border-left-width, border-radius, margin, padding